### PR TITLE
Bugfix/schema alignment

### PR DESCRIPTION
**Pull request by jenna-anderson**
## Description

**See corresponding bugfix/schema-alignment PR in frontend for reference: [Frontend PR](https://github.com/Lambda-School-Labs/human-rights-first-asylum-fe-a/pull/352) **

We aligned schema variable names from the data science database to the backend. The schema names were as follows:

- date to decision_date
- type_of_violence to type_of_persecution
- credible to credibility
- filed_in_one_year to check_for_one_year

This was done to ensure that the web backend and data science backend could be swapped out with ease. We worked with frontend to make sure nothing was broken after the changes.
[Trello Card](https://trello.com/c/UbRXBqZ5)

Note: The ds_case seed file was also deleted because it was obsolete and breaking running seeds (Alex Tran from the last cohort informed us it was unnecessary) 

Contributors to this pull request:
@ZacharyCooremans @joshuascan @schandrupatla @TrevorDewalt 

Fixes # (issue)

This fix allows the frontend to work with either database through having consistent variable names.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

https://user-images.githubusercontent.com/81700660/132600058-c20d15bf-994c-4391-9859-ce679ac3a455.mp4

### CODE REVIEW

**Review by ctscofield — APPROVED** [review]
Very good attention to detail with updating the variable names. I can't find any flaws.

**Review by RickyKlusmeier — APPROVED** [review]
This all looks good to me, all the right places were changed to the new schema

File: README.md [hunk 1]
```diff
@@ -51,10 +51,10 @@ The base technologies are JavaScript, HTML and CSS. The frontend leverages [Reac
         "gender": "Male",
         "applicant_language": "Spanish",
         "indigenous_group": "Not Applicable",
-        "type_of_violence": "Not Applicable",
+        "type_of_persecution": "Not Applicable",
         "appellate": false,
-        "filed_in_one_year": false,
-        "credible": true,
+        "check_for_one_year": false,
+        "credibility": true,
```
Comment (paulstgermain) [inline README.md:57]: Good attention to detail in editing the README as we changed these variables - the importance of up-to-date documentation cannot be understated!

File: data/migrations/20210821094636_ds_case.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ exports.up = function (knex) {
   //   table.string('case_origin_state');
   //   table.string('case_origin_city');
   //   table.string('protected_grounds');
-  //   table.string('type_of_violence');
+  //   table.string('type_of_persecution');
```
Comment (paulstgermain) [inline data/migrations/20210821094636_ds_case.js:14]: Let's make a note to review the need for the `ds_case` migration and seeds and come to a decision about keeping it or not.

**Comment by chrisjs87** [comment]
I went through and made sure that everything changed is consistent. I did not see any discrepancies. Good job everyone!

**Review by chrisjs87 — APPROVED** [review]
I went through and made sure that everything changed is consistent. I did not see any discrepancies. Good job everyone!

**Review by paulstgermain — APPROVED** [review]
## Video Walkthrough Notes

### Sri

- I really appreciate that you took time to start your walkthrough of this PR off by showing the task on the Trello board that you sought to complete with this PR and then describing what the problem was before diving into the changes made.
- I do feel that you could have spent more time making it clear that the data sent back from DS's scraper wasn't matching the shape of the data we were expecting on the front and back ends, but your explanation was still understandable, so this won't block the PR. Remember when talking through your work - more detail!
- You made it clear that your changes were very well tested, and we all know how important perform real tests of our code is - well done!
- You also spoke through the technical aspects of the work you did here very competently.

### Jenna

- You immediately built off of the work Sri presented very competently and thoroughly, and I love that you mentioned some of this work came from Stakeholder feedback - getting used to talking about your experience working with stakeholders is going to be a huge help to you when interviewing for new positions!
- You also made great use of the PR's changes page to showcase the changes you made, and I appreciated how clear that made the work done here.

## Code Review

This is a fairly straightforward change to the codebase - by changing the expected shape of the data to be received from the DS API and making sure we're expecting the appropriate variable names as well as changing our database schema, we're taking steps toward realigning the flow of data in our application - this is a huge deal!

I appreciate the level of consistency here in regards to there not being any changes in the structure of the code changed and the dedication to consistently using single quotes. Not seeing any leftover comments or unused code here, so I'm happy to approve and merge this PR into the codebase!

**Pull request merged**